Sasquatch/Sasquatch/AppCenterDelegate.swift

```diff
@@ -16,7 +16,6 @@ import AppCenterAnalytics
   func setCustomProperties(_ customProperties: MSCustomProperties)
   func installId() -> String
   func appSecret() -> String
-  func logUrl() -> String
```

Comment:
Is this change intentionally part of the PR?

Reply:
No use in the test app. We don't read the log url from SDK in the test app now.